### DISCUSSION TITLE
Add a grace period to the accuracy challenge mod

### DISCUSSION THREAD

**Discussion by w4536p** [ppy/osu#23592] [Ideas]
I think that adding a grace period to the accuracy challenge mod would make it more usable and less frustrating to use as any mistake before the first 10 circles will instantly fail you on any minimum accuracy >= 90%, ideally there should be an option to add a grace period of x circles where the accuracy is not taken into account so that a small mistake such a hitting a 100 is not going to fail you instantly, I tried to do this myself but I do not have the knowledge needed to implement this but I will still attach my attempt at implementing it as some sort of suggestion for how it can be implemented  
[ModAccuracyChallenge.cs.gz](https://github.com/ppy/osu/files/11513527/ModAccuracyChallenge.cs.gz)

**Comment by monochrome22** [comment]
This wouldn't be necessary if the accuracy challenge mod was based on maximum accuracy instead, which is being PRed https://github.com/ppy/osu/pull/23317.
